Sample-Stack Throwing Test Error

I am running from command line gradelew appserver. java version 1.7 64 bit windows 7
:compileTestGroovy UP-TO-DATE
:processTestResources UP-TO-DATE
:testClasses
:test

com.marklogic.samplestack.database.DatabaseQnADocumentSearchIT > testAcceptedSea
rch                                                                            F
AILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationExcepti
on
                    Caused by: org.springframework.beans.factory.BeanCreationExc
eption
                        Caused by: org.springframework.beans.factory.BeanCreatio
nException
                            Caused by: com.marklogic.client.FailedRequestExcepti
on

:buildSrc:compileJava UP-TO-DATE
:buildSrc:compileGroovy UP-TO-DATE
:buildSrc:processResources UP-TO-DATE
:buildSrc:classes UP-TO-DATE
:buildSrc:jar UP-TO-DATE
:buildSrc:assemble UP-TO-DATE
:buildSrc:compileTestJava UP-TO-DATE
:buildSrc:compileTestGroovy UP-TO-DATE
:buildSrc:processTestResources UP-TO-DATE
:buildSrc:testClasses UP-TO-DATE
:buildSrc:test UP-TO-DATE
:buildSrc:check UP-TO-DATE
:buildSrc:build UP-TO-DATE
:compileJava UP-TO-DATE
:compileGroovy UP-TO-DATE
:processResources UP-TO-DATE
:classes UP-TO-DATE
:jar
:bootRepackage
:dbConfigureClean
:dbInit
Server already secured.  Initialization skipped.
Server already secured.  Initialization skipped
Creating users and roles...
[errorResponse:[message:MANAGE-CONFLICTINGCONFIG: (err:FOER0000) A conflict has been detected:  User exists., messageCode:MANAGE-CONFLICTINGCONFIG, status:Bad Request, statusCode:400]]
[errorResponse:[message:MANAGE-CONFLICTINGCONFIG: (err:FOER0000) A conflict has been detected:  User exists., messageCode:MANAGE-CONFLICTINGCONFIG, status:Bad Request, statusCode:400]]
[errorResponse:[message:MANAGE-CONFLICTINGCONFIG: (err:FOER0000) A conflict has been detected:  User exists., messageCode:MANAGE-CONFLICTINGCONFIG, status:Bad Request, statusCode:400]]
[errorResponse:[message:Your bootstrap payload caused the server to throw an error.  Underlying error message: REST-INVALIDPARAM: (err:FOER0000) Invalid parameter: Port 8006 is in use., messageCode:RESTAPI-INVALIDCONTENT, status:Bad Request, statusCode:400]]
:dbConfigure
Putting database configuration database-properties.json
Success: 204
Looks like xxx\marklogic-samplestack-master\database\LICENSE is not a MarkLogic configuration file... skipping
Looks like xxx\marklogic-samplestack-master\database\README.md is not a MarkLogic configuration file... skipping
Success: 204
Putting search options contributors.json
Success: 204
Putting search options questions.json
null
Putting search options single-question.json
Success: 204
Putting search options tags.json
null
Putting service extension relatedTags.sjs
Success: 204
Putting transform search-response.sjs
Success: 204
Putting transform single-question.sjs
Success: 204
:props UP-TO-DATE
:compileTestJava UP-TO-DATE
:compileTestGroovy UP-TO-DATE
:processTestResources UP-TO-DATE
:testClasses UP-TO-DATE
:test

com.marklogic.samplestack.database.DatabaseQnADocumentSearchIT > testAcceptedSearch FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.database.DatabaseQnADocumentSearchIT > guestSearchSeesOnlyResolvedQuestions FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.database.DatabaseQnADocumentSearchIT > testResponseExtracts FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.database.DatabaseQnADocumentSearchIT > testConfiguredStringSearches FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.database.DatabaseQnADocumentSearchIT > testSingleGetTransform FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.database.DatabaseQnADocumentSearchIT > defaultSearchOrdersByActivityDescending FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.database.DatabaseQnADocumentSearchIT > testActivitySearch FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.database.DatabaseQnADocumentSearchIT > authenticatedSearchSeesUnresolvedQuestions FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.database.DocumentSecurityIT > testDocumentSecurity FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.ContributorServiceIT > testContributorCRUD FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.ContributorServiceIT > testUserNameCardinality FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.QnAServiceIT > testAskAndAnswerFlow FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.QnAServiceIT > testCRUD FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.QnAServiceIT > testCommentsAndSearch FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.QnAServiceIT > testVoting FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.TagsServiceIT > testStructuredTagSearch FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.TagsServiceIT > testTagTwoSearch FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.TagsServiceIT > testAllTagValues FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.TagsServiceIT > testTagSearchWithForTag FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.ContributorControllerIT > testContributorCRUD FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.LDAPIT > testFinds FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.LoginIT > loginBadCredentials FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.LoginIT > loginForbidden FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.LoginIT > loginNormalFlow FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testAcceptAnswer FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testAnswerQuestion FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testVoteUpAnswer FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > commentOnAnswer FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testAskQuestion FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testAnonymousAccessToAccepted FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testVoteDownQuestion FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testIncludeTimezoneAdjustsDateFacet FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testAnonymousCanSearch FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testAskMalformedQuestions FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testVoteDownAnswer FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testVoteUpQuestion FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testAnonymousCannotAsk FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testLoggedInCanSearch FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.TagControllerIT > testTagsWithPageLength FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.TagControllerIT > testTagsNoArgs FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.TagControllerIT > testBadSort FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.TagControllerIT > testStartLimitOrder FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.TagControllerIT > testRelatedTags FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.TagControllerIT > testTagsAnonymousOK FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.TagControllerIT > testTagsWithArgument FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.TagControllerIT > testSortFrequency FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException
19:44:31.935 [Thread-32] INFO  o.s.w.c.s.GenericWebApplicationContext - Closing org.springframework.web.context.support.GenericWebApplicationContext@209cccc1: startup date [Wed Feb 11 19:44:30 PST 2015]; root of context hierarchy
19:44:31.935 [Thread-31] INFO  o.s.w.c.s.GenericWebApplicationContext - Closing org.springframework.web.context.support.GenericWebApplicationContext@24f98ad2: startup date [Wed Feb 11 19:44:27 PST 2015]; root of context hierarchy
19:44:31.940 [Thread-32] INFO  o.s.j.e.a.AnnotationMBeanExporter - Unregistering JMX-exposed beans on shutdown
:test FAILED

BUILD FAILED

Total time: 1 mins 2.353 secs

it is same please see the log now.
:buildSrc:compileJava UP-TO-DATE
:buildSrc:compileGroovy UP-TO-DATE
:buildSrc:processResources UP-TO-DATE
:buildSrc:classes UP-TO-DATE
:buildSrc:jar UP-TO-DATE
:buildSrc:assemble UP-TO-DATE
:buildSrc:compileTestJava UP-TO-DATE
:buildSrc:compileTestGroovy UP-TO-DATE
:buildSrc:processTestResources UP-TO-DATE
:buildSrc:testClasses UP-TO-DATE
:buildSrc:test UP-TO-DATE
:buildSrc:check UP-TO-DATE
:buildSrc:build UP-TO-DATE
:compileJava
:compileGroovy UP-TO-DATE
:processResources
:classes
:jar
:bootRepackage
:dbConfigureClean
:dbInit
Server already secured.  Initialization skipped.
Server already secured.  Initialization skipped
Creating users and roles...
Success: 201
:dbConfigure
Putting database configuration database-properties.json
Success: 204
Looks like xxx\marklogic-samplestack-master\database\LICENSE is not a MarkLogic configuration file... skipping
Looks like xxx\marklogic-samplestack-master\database\README.md is not a MarkLogic configuration file... skipping
Success: 204
Putting search options contributors.json
Success: 201
Putting search options questions.json
null
Putting search options single-question.json
Success: 201
Putting search options tags.json
null
Putting service extension relatedTags.sjs
Success: 204
Putting transform search-response.sjs
Success: 204
Putting transform single-question.sjs
Success: 204
:props UP-TO-DATE
:compileTestJava
:compileTestGroovy UP-TO-DATE
:processTestResources
:testClasses
:test

com.marklogic.samplestack.database.DatabaseQnADocumentSearchIT > testAcceptedSearch FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.database.DatabaseQnADocumentSearchIT > guestSearchSeesOnlyResolvedQuestions FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.database.DatabaseQnADocumentSearchIT > testResponseExtracts FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.database.DatabaseQnADocumentSearchIT > testConfiguredStringSearches FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.database.DatabaseQnADocumentSearchIT > testSingleGetTransform FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.database.DatabaseQnADocumentSearchIT > defaultSearchOrdersByActivityDescending FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.database.DatabaseQnADocumentSearchIT > testActivitySearch FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.database.DatabaseQnADocumentSearchIT > authenticatedSearchSeesUnresolvedQuestions FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.database.DocumentSecurityIT > testDocumentSecurity FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.ContributorServiceIT > testContributorCRUD FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.ContributorServiceIT > testUserNameCardinality FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.QnAServiceIT > testAskAndAnswerFlow FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.QnAServiceIT > testCRUD FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.QnAServiceIT > testCommentsAndSearch FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.QnAServiceIT > testVoting FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.TagsServiceIT > testStructuredTagSearch FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.TagsServiceIT > testTagTwoSearch FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.TagsServiceIT > testAllTagValues FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.service.TagsServiceIT > testTagSearchWithForTag FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.ContributorControllerIT > testContributorCRUD FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.LDAPIT > testFinds FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.LoginIT > loginBadCredentials FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.LoginIT > loginForbidden FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.LoginIT > loginNormalFlow FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testAcceptAnswer FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testAnswerQuestion FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testVoteUpAnswer FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > commentOnAnswer FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testAskQuestion FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testAnonymousAccessToAccepted FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testVoteDownQuestion FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testIncludeTimezoneAdjustsDateFacet FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testAnonymousCanSearch FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testAskMalformedQuestions FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testVoteDownAnswer FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testVoteUpQuestion FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testAnonymousCannotAsk FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.QnADocumentControllerIT > testLoggedInCanSearch FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.TagControllerIT > testTagsWithPageLength FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.TagControllerIT > testTagsNoArgs FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.TagControllerIT > testBadSort FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.TagControllerIT > testStartLimitOrder FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.TagControllerIT > testRelatedTags FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.TagControllerIT > testTagsAnonymousOK FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.TagControllerIT > testTagsWithArgument FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException

com.marklogic.samplestack.integration.web.TagControllerIT > testSortFrequency FAILED
    java.lang.IllegalStateException
        Caused by: org.springframework.beans.factory.BeanCreationException
            Caused by: org.springframework.beans.factory.BeanCreationException
                Caused by: org.springframework.beans.factory.BeanCreationException
                    Caused by: org.springframework.beans.factory.BeanCreationException
                        Caused by: org.springframework.beans.factory.BeanCreationException
                            Caused by: com.marklogic.client.FailedRequestException
20:05:18.994 [Thread-31] INFO  o.s.w.c.s.GenericWebApplicationContext - Closing org.springframework.web.context.support.GenericWebApplicationContext@25c4eb0a: startup date [Wed Feb 11 20:05:14 PST 2015]; root of context hierarchy
20:05:19.006 [Thread-32] INFO  o.s.w.c.s.GenericWebApplicationContext - Closing org.springframework.web.context.support.GenericWebApplicationContext@69907472: startup date [Wed Feb 11 20:05:17 PST 2015]; root of context hierarchy
20:05:19.007 [Thread-32] INFO  o.s.j.e.a.AnnotationMBeanExporter - Unregistering JMX-exposed beans on shutdown
:test FAILED

BUILD FAILED

Total time: 1 mins 14.439 secs

would this help.. 
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'markLogicContributorService': Invocation of init method failed; nested exception is com.marklogic.client.FailedRequestException: Local message: search failed: Bad Request. Server Message: REST-INVALIDPARAM: (err:FOER0000) Invalid parameter: Invalid query: XDMP-VALIDATEUNEXPECTED: (err:XQDY0027) validate strict { $q } -- Invalid node: Found @type but expected () at fn:doc("")/search:query/search:value-query/@type using schema "search.xsd"
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:136)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyBeanPostProcessorsBeforeInitialization(AbstractAutowireCapableBeanFactory.java:408)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1558)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:539)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:476)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:303)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:299)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:194)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.findAutowireCandidates(DefaultListableBeanFactory.java:1127)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1051)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:949)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:533)
	... 75 more
Caused by: com.marklogic.client.FailedRequestException: Local message: search failed: Bad Request. Server Message: REST-INVALIDPARAM: (err:FOER0000) Invalid parameter: Invalid query: XDMP-VALIDATEUNEXPECTED: (err:XQDY0027) validate strict { $q } -- Invalid node: Found @type but expected () at fn:doc("")/search:query/search:value-query/@type using schema "search.xsd"
	at com.marklogic.client.impl.JerseyServices$JerseySearchRequest.getResponse(JerseyServices.java:2206)
	at com.marklogic.client.impl.JerseyServices.getBulkDocumentsImpl(JerseyServices.java:869)
	at com.marklogic.client.impl.JerseyServices.getBulkDocuments(JerseyServices.java:744)
	at com.marklogic.client.impl.DocumentManagerImpl.search(DocumentManagerImpl.java:500)
	at com.marklogic.client.impl.JSONDocumentImpl.search(JSONDocumentImpl.java:26)
	at com.marklogic.client.impl.PojoRepositoryImpl.search(PojoRepositoryImpl.java:340)
	at com.marklogic.client.impl.PojoRepositoryImpl.search(PojoRepositoryImpl.java:317)
	at com.marklogic.samplestack.dbclient.MarkLogicContributorService.getByUserName(MarkLogicContributorService.java:148)
	at com.marklogic.samplestack.dbclient.MarkLogicContributorService.store(MarkLogicContributorService.java:178)
	at com.marklogic.samplestack.dbclient.MarkLogicContributorService.store(MarkLogicContributorService.java:173)
	at com.marklogic.samplestack.dbclient.MarkLogicContributorService.storeJoeAndMary(MarkLogicContributorService.java:109)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleElement.invoke(InitDestroyAnnotationBeanPostProcessor.java:349)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMetadata.invokeInitMethods(InitDestroyAnnotationBeanPostProcessor.java:300)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:133)
	... 87 more

--- a/appserver/java-spring/buildSrc/src/main/groovy/MarkLogicConfigureTask.groovy
+++ b/appserver/java-spring/buildSrc/src/main/groovy/MarkLogicConfigureTask.groovy
@@ -97,8 +97,7 @@ public class MarkLogicConfigureTask extends MarkLogicTask {
 
     void putTransform(transform) {
         def transformFileName = transform.getPath().replaceAll(~"\\\\","/")
-        def transformName = transformFileName.replaceAll(~"\\.[^\\.]+", "").replaceAll(~".*\\/","")
-
+        def transformName = transformFileName.split("/")[-1].replaceAll(~"\\.[^\\.]+", "")
         if (transformName) {
             logger.info( "Saving transform " + transformName)
             RESTClient client = new RESTClient("http://" + config.marklogic.rest.host + ":" + config.marklogic.rest.port + "/v1/config/transforms/" + transformName)

